### PR TITLE
update base instructions

### PR DESCRIPTION
**Pull request by aditya-manit**

### CODE REVIEW

**Review by blockchainguyy — APPROVED** [review]

**Review by ffe9f8 — APPROVED** [review]

File: src/pages/validator/external-chains/base.mdx [hunk 1]
```diff
@@ -33,8 +33,22 @@ snap install docker
...
+```bash
+    env_file:
+      # select your network here:
+#      - .env.sepolia
+#      - .env.mainnet
+```
+Update the following variables in your `.env` file
+```bash
+# [recommended] replace with your preferred L1 (Ethereum, not Base) node RPC URL:
+OP_NODE_L1_ETH_RPC=https://rpc.sepolia.org
```
Comment (milapsheth) [inline src/pages/validator/external-chains/base.mdx:47]:
```suggestion
# Set the RPC url for your own L1 (Ethereum) node:
OP_NODE_L1_ETH_RPC=[your ethereum RPC]
```

File: src/pages/validator/external-chains/base.mdx [hunk 2]
```diff
@@ -33,8 +33,22 @@ snap install docker
...
+
+# [required] replace with your preferred L1 CL beacon endpoint:
+OP_NODE_L1_BEACON=https://your.sepolia.beacon.node/endpoint-here
```
Comment (milapsheth) [inline src/pages/validator/external-chains/base.mdx:50]:
```suggestion
# Set the url for your own L1 CL beacon endpoint:
OP_NODE_L1_BEACON=[your beacon node]
```

File: src/pages/validator/external-chains/base.mdx [hunk 3]
```diff
@@ -33,8 +33,22 @@ snap install docker
...
 
-2. In `docker-compose.yml`, set `OP_NODE_L1_ETH_RPC` to your own Goerli L1 RPC node. Using your own Goerli node is required to guarantee decentralization for validation.
+2. In `docker-compose.yml`, uncomment the right .env file according to the network you want to run it on
 
+```bash
+    env_file:
+      # select your network here:
+#      - .env.sepolia
+#      - .env.mainnet
+```
```
Comment (milapsheth) [inline src/pages/validator/external-chains/base.mdx:43]: This impacts Optimism as well (and potentially Mantle if you can take a look)

File: src/pages/validator/external-chains/base.mdx (hunk 2 above)
Comment (milapsheth) [inline src/pages/validator/external-chains/base.mdx:50]: Does the Ethereum setup indicate how to get the beacon endpoint? If not, can you add some instructions to it, and link to that here?

Comment (aditya-manit) [reply src/pages/validator/external-chains/base.mdx]: I think docs should have it already, will add a link here

File: src/pages/validator/external-chains/base.mdx (hunk 3 above)
Comment (aditya-manit) [reply src/pages/validator/external-chains/base.mdx:43]: Yes, will update other l2 too in a different PR

File: src/pages/validator/external-chains/base.mdx (hunk 2 above)
Comment (aditya-manit) [reply src/pages/validator/external-chains/base.mdx:55]: Added

**Comment by aditya-manit** [comment]
I dont think base PR was done by us, so I have update it all to match the format of other docs

**Pull request merged**